fcrepo4-exts/fcrepo-camel-toolbox
# Changes to work with Fedora-Camel 5.0.0

**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2787

Related PR: https://github.com/fcrepo4-exts/fcrepo-camel/pull/150

# What does this Pull Request do?
Most of this is @dbernstein's work from before, I just dragged it across a line.

# What's new?
I checked out a bunch of things from @dbernstein's update-fcrepo-parent-pom branch but stuck with the older versions of dependencies. 

# How should this be tested?

Should build and then you could try testing it out. I am going to try it in fcrepo-vagrant soon.

# Additional Notes:

Example:
* Does this change require documentation to be updated? no
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# Interested parties
@peichman-umd @birkland @dbernstein or any other interested @fcrepo4/committers
